docs/competition/volunteers/refreshments.md

```diff
@@ -2,27 +2,31 @@
 
 It's important that volunteers stay hydrated, which can be difficult for those in roles which don't allow them to move. (Don't give refreshments to competitors).
 
-**In SR2019, the refreshments are in the Volunteer Rest room, which is on level 2 past the toilets and to the left. Also water points are on both level 2 and 3 in the corner furthest from the toilets.**
+**In SR2022, the refreshments are in the Volunteer Rest room, which is on level 2 past the toilets and to the left. Also water points are on both level 2 and 3 in the corner furthest from the toilets.**
 
 It is the responsibility of the refreshments role to offer and provide refreshments to all volunteers, especially those who can't leave their roles.
 
 To do this, you should grab a tray, fill it with a mix of drinks and food, and go to all volunteer positions, offering refreshments.
 Please also fill some cups with water from a nearby water fountain, and brew coffee/tea if available.
 
-People at the following locations will be unable to leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
+People at the following locations are usually unable to leave their posts to fetch refreshments for themselves, so it is important to take refreshments to them regularly:
 
-- tech desks
-- reception
-- helpdesk
-- arena: commentator, marshals, head shepherd, score entry
+- Tech desks
+- Reception
+- Helpdesk
+- Arena: commentator, marshals, head shepherd, score entry
 
 The tech desks roles and commentator are particularly important as they do not rotate between people at all.
 
+## Immediately after Lunch
+
+If your role is after lunch, it's very important to make sure everyone has had the time to eat food. If they haven't eaten and don't seem like they have the time to, give them a sandwich from our emergency sandwich supply.
+
 ## Running low on refreshments
 
 If we're running low on refreshments, it's your responsibility to make sure someone gets more from the nearby shops. Find someone or speak to the volunteer manager to find someone who is free to go get more. There should be cash by the refreshment storage for you to buy more. It's intended to last for the whole competition, so do not spend it all. If the money has ran out, please speak to the Volunteer Manager.
```

Comment:
Should volunteer manager be volunteer coordinator (or event coordinator as they are the same person this year)?